prism/src/main/java/org/apache/falcon/resource/AbstractInstanceManager.java

```diff
@@ -676,6 +683,79 @@ public TriageResult triageInstance(String entityType, String entityName, String
         }
     }
 
+    public InstancesResult searchInstances(String type, String nameSubsequence, String tagKeywords,
```

Comment:
searchInstances function is retrieving and populating entityVertices, adding and running query to get instanceMap, populating instances List with search result. These tasks are happening in searchInstances function along with calling of sortInstancesPagination() and returning InstanceResult as output.  So I thought it is better to split the searchInstances function with mentioned tasks in subfunction for better manageability going forward.